INSTALL.md

```diff
@@ -49,21 +49,11 @@ Prerequisites
 
         sudo apt-get install git-core
 
-5.  Memcached
-
-    By default the MusicBrainz server requires a Memcached server running on the
-    same server with default settings. To install Memcached, run the following:
-
-        sudo apt-get install memcached
-
-    You can change the memcached server name and port, or configure other datastores
-    in lib/DBDefs.pm.
-
 6.  Redis
```

Comment:
Yes, the Markdown definition of a numbered list is essentially `<any digits>. `. So `1. foo 1. bar 1. baz` is mostly equivalent to HTML `<ol><li>foo</li> <li>bar</li> <li>baz</li>`, generating its own numbering.

Still, the point of MD is to be also readable without rendering, so I agree it should be fixed. :-)